db/migrate/20240219121949_create_needed_item.rb

```diff
@@ -0,0 +1,9 @@
+class CreateNeededItem < ActiveRecord::Migration[7.0]
+  def change
+    create_table :needed_items do |t|
+      t.integer :child_id, null: false, index: true
+      t.integer :item_id, null: false, index: true
```

Comment:
These should be references rather than just integers.

Reply:
addressed